app/assets/stylesheets/views/_home-thumbnail-grid.css.scss

```diff
@@ -45,3 +45,33 @@ $home-thumbnail-grid-author: $home-thumbnail-grid-avatar + ($home-thumbnail-grid
 
   @include ellipsis;
 }
+
+.home-fluid-thumbnail-grid-details {
```

Comment:
These "details" marked styles are separated from existing styles, so that css compilation should not be a problem:
.home-fluid-thumbnail-grid-details
.home-fluid-thumbnail-grid-details-author-name
..home-fluid-thumbnail-grid-details-distance